integration/tests/test_sanity.py

```diff
@@ -18,6 +18,7 @@
     install,
     spin,
     uninstall,
+    get_and_verify_plan,
```

Comment:
Undo this change? I'd assume it was left over from earlier work on this PR. Function doesn't appear to be used in test_sanity.